server_manager/www/ui_components/outline-server-view.ts

```diff
@@ -922,6 +958,33 @@ export class ServerView extends DirMixin(PolymerElement) {
     return this.accessKeyRows.find(key => key.id === id);
   }
 
+  _computeServerMetrics(
+    totalDevices: number,
+    totalUserHours: number,
+    totalInboundBytes: number,
+    language: string
+  ) {
+    return [
+      {
+        icon: 'devices',
+        name: this.localize('server-metrics-average-devices'),
+        value: totalDevices.toFixed(2),
+      },
+      {
+        icon: 'timer',
+        name: this.localize('server-metrics-user-hours'),
+        units: this.localize('server-metrics-user-hours-unit'),
```

Comment:
This is not really how we should be formatting units. Use the standard libraries instead: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#unit_2